nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StartDomainCommand.java

```diff
@@ -143,7 +143,18 @@ protected int executeCommand() throws CommandException {
                 logger.fine(Strings.get("dry_run_msg"));
                 List<String> cmd = glassFishLauncher.getCommandLine();
                 StringBuilder sb = new StringBuilder();
+                boolean skipLine = false;
```

Comment:
just an idea that does not use the skip flag :)
```java
        int indexOfReadStdin = cmd.indexOf("-read-stdin");
        List cmdForRun = IntStream.range(0, cmd.size())
             .filter(index -> index < indexOfReadStdin || index > indexOfReadStdin + 1)
             .mapToObj(cmd::get).collect(Collectors.toList());
```

Reply:
Thanks for your suggestion. I refactored the code as you suggested.